### PR TITLE
gh-25 - Updated gaffer to 0.4.7

### PR DESCRIPTION
**Pull request by p013570**

### CODE REVIEW

File: graffer-ql/pom.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (t616178) [inline graffer-ql/pom.xml]: Does this indicate that we've got another issue with the shared code formatting settings?

File: graffer-ql/pom.xml [hunk 2]
```diff
@@ -28,7 +29,7 @@
         </dependency>
         <dependency>
             <groupId>gaffer</groupId>
-            <artifactId>example</artifactId>
+            <artifactId>example-graph</artifactId>
```
Comment (t616178) [inline graffer-ql/pom.xml]: Was there a "migration-required" note for this change?

File: graffer-ql/pom.xml (hunk 1 above)
Comment (p013570) [inline graffer-ql/pom.xml]: Yes...

File: graffer-ql/pom.xml (hunk 2 above)
Comment (p013570) [inline graffer-ql/pom.xml]: I've added a note to the Gaffer issue that made the change.

**Pull request merged**